docs/content/docs/testing/telemetry.md

```diff
@@ -240,24 +236,55 @@ export class ConsoleLogger implements ITelemetryBaseLogger {
-object. The custom logger is now hooked up to the container's telemetry system.
+This custom logger should be provided in the service client constructor and will be created when creating or getting
+the container. Once the container is returned, the custom logger is now hooked up to the container's telemetry system.
 
 ```ts
 async function start(): Promise<void> {
-    // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
-    // and hook to the Telemetry system
-    const consoleLogger: ConsoleLogger = new ConsoleLogger();
-
-    // Get or create the document depending if we are running through the create new flow
-    const client = useAzure ? AzureClient :  new TinyliciousClient();
-    const [fluidContainer, containerServices] = createNew
-        ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
-        : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
+  // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
+  // and hook to the Telemetry system
+  const azureConfig = {
+      connection: connectionProps,
+      logger: new ConsoleLogger(),
+  };
+  const client = new AzureClient(azureConfig);
+  let container: FluidContainer;
+  let services: AzureContainerServices;
+
+  // Get or create the document depending if we are running through the create new flow
+  const createNew = !location.hash;
+  if (createNew) {
+      // The client will create a new detached container using the schema
+      // A detached container will enable the app to modify the container before attaching it to the client
+      {container, services} = await client.createContainer(containerSchema);
+  }
 ```
 
 Now, whenever a telemetry event is encountered, the custom `send()` method gets called and will print out the entire
 event object.
 
 <img src="https://fluidframework.blob.core.windows.net/static/images/consoleLogger_telemetry_in_action.png" alt="The
   ConsoleLogger sends telemetry events to the browser console for display.">
+
+{{% callout warning %}}
+
+The purpose of `ConsoleLogger` is purely to demonstrate how the `ITelemetryBaseLogger` interface should be implemented. The
+Fluid Framework provides `DebugLogger` by default if a custom logger is not provided. For more information, please refer to
+[Using DebugLogger](#using-debuglogger) below instead of implementing something similar to `ConsoleLogger`.
+
+{{% /callout %}}
+
+### Using `DebugLogger`
+
+The `DebugLogger` offers a convenient way to output all telemetry events to the console. Like mentioned previously, creating
+or getting a container creates a `Loader` object, which then also provides a `DebugLogger` as a mixin in its constructor. Hence,
+`DebugLogger` is already present by default when creating/getting a container, and no extra steps are required to use it.
```

Comment:
```suggestion
The `DebugLogger` offers a convenient way to output all telemetry events to the console. `DebugLogger` is present by default when creating/getting a container, and no extra steps are required to use it.
```

I think the details of the code flow aren't relevant, so just focus on the usage.

Reply:
I agree, corresponding to the previous comment you mentioned above.